includes/admin/class-wc-payments-admin.php

```diff
@@ -26,15 +26,28 @@ class WC_Payments_Admin {
 	 */
 	private $account;
 
+	/**
+	 * Client for making requests to the WooCommerce Payments API
+	 *
+	 * @var WC_Payments_API_Client
+	 */
+	private $payments_api_client;
+
 	/**
 	 * Hook in admin menu items.
 	 *
 	 * @param WC_Payment_Gateway_WCPay $gateway WCPay Gateway instance to get information regarding WooCommerce Payments setup.
 	 * @param WC_Payments_Account      $account Account instance.
+	 * @param WC_Payments_API_Client   $payments_api_client - WooCommerce Payments API client.
 	 */
-	public function __construct( WC_Payment_Gateway_WCPay $gateway, WC_Payments_Account $account ) {
-		$this->wcpay_gateway = $gateway;
-		$this->account       = $account;
+	public function __construct(
+		WC_Payment_Gateway_WCPay $gateway,
+		WC_Payments_Account $account,
+		WC_Payments_API_Client $payments_api_client
+	) {
+		$this->wcpay_gateway       = $gateway;
+		$this->account             = $account;
+		$this->payments_api_client = $payments_api_client;
```

Comment:
I can't find any usage of `$payments_api_client`, probably it's a left over.

Reply:
Yes, it was a left-over and is now gone.